meraki_appliance.tf

```diff
@@ -519,7 +519,7 @@ locals {
           key           = format("%s/%s/%s", domain.name, organization.name, network.name)
           network_id    = meraki_network.organizations_networks[format("%s/%s/%s", domain.name, organization.name, network.name)].id
           vlans_enabled = try(length(network.appliance.vlans) > 0, false)
-        } if try(length(network.appliance.vlans) > 0, false)
+        } if try(network.appliance, null) != null
```

Comment:
@jon-humphries under what condition should the `networks_appliance_vlans_settings` resource be created?

Originally, it was always created; then #47 changed it to check `network.appliance.vlans` (i.e. to create a resource only if a VLAN is configured, i.e. only with `vlans_enabled = true`.